src/main/java/labs/introtoprogramming/lab5/object/Parser/ParseObjFile.java

```diff
@@ -0,0 +1,155 @@
+package labs.introtoprogramming.lab5.object.Parser;
+
+import labs.introtoprogramming.lab5.geometry.Vector3;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.logging.Logger;
+
+import static java.util.logging.Level.INFO;
+import static java.util.logging.Level.WARNING;
+
+public class ParseObjFile {
+    private Logger log = Logger.getLogger(ParseObjFile.class.getName());
+
+    /**
+     * String indicators of parameter in .obj file.
+     */
+
+    private final static String OBJ_VERTEX_NORMAL = "vn";
+    private final static String OBJ_VERTEX = "v";
+    private final static String OBJ_FACE = "f";
+
+    private ArrayList<Vector3> verticesNormals = new ArrayList<>();
+    private ArrayList<Vector3> verticesGeometry = new ArrayList<>();
+    private PolygonObject polygonObject = null;
+
+    File objFile = null;
+
+
+    public ParseObjFile(String filename) throws FileNotFoundException, IOException {
+
+        polygonObject = new PolygonObject();
+        parseObjFile(filename);
+
+    }
+
+    /**
+     * This part pars object file.
+     *
+     * @param objFilename
+     * @throws FileNotFoundException
+     * @throws IOException
+     */
+    private void parseObjFile(String objFilename) throws FileNotFoundException, IOException {
```

Comment:
It is better to make this method accept `InputStream` and add another overloaded method which passes FileInputStream.